src/BlazorUI/Bit.BlazorUI/Components/Surfaces/Card/BitCard.scss

```
@@ -1,182 +1,468 @@
+    > .bit-crd-img {
+        height: var(--bit-crd-img-height, 100%);
+    }
 }
 
-.bit-crd-sbr {
-    border-color: $clr-brd-sec;
+// Behind rather than above: the cover is taken out of the flow and stretched over the whole surface, and the
+// column of content is written on top of it. It is declared after the horizontal block so it wins over the
+// share of the width that one hands the cover - an absolutely positioned box with a left, a right and a width
+// drops one of the three, and the one it drops is not the width.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)